dev/com.ibm.ws.jpa.tests.container.client_fat/test-applications/BasicJPAFieldInjectClient.jar/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence 
+    xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd"
+    version="2.1">
+
+    <persistence-unit name="AppCliPU_1">
+        <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+        <non-jta-data-source>jdbc/JPA_NJTADS</non-jta-data-source>
+        <class>com.ibm.ws.jpa.clientcontainer.fat.basic.fieldinj.entities.SimpleEntity</class>
+        
+        <properties>
+            <!-- EclipseLink should create the database schema automatically -->
+            <property name="eclipselink.ddl-generation" value="create-tables" />
```

Comment:
Are we not using the DDL generation pattern our other FATs use?

If this was a conscious decision, can we instead use:
```xml
    <property name="javax.persistence.schema-generation.database.action" value="drop-and-create"/>
    <property name="jakarta.persistence.schema-generation.database.action" value="drop-and-create"/>
```

Reply:
For these, no.  This is a quick port of a number of small tests.  I decided not to modify them to use the ddl servlet.  Sure, I can adopt drop-and-create.